<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {	font-family: &quot;微软雅黑&quot;; font-size: 15px;}
QLabel#labelLogo {
	background: #f3f3f3;
	outline:none;
	color:#333;
}


QPushButton#btnMin, QPushButton#btnClose,QPushButton#btnConf {
	background: transparent;
	border: none;
	color: #000;
	font-size: 18px;
}


QPushButton#btnMin:hover, QPushButton#btnConf:hover {
	background: #af1d1d1d;
}

QPushButton#btnClose:hover {
	background: red;
}

QPushButton#btnLogin {
	border: 1px solid #c9c9c9;
	background: #569082;
	color:#ffffff;
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
}

QLabel#labelId, QLabel#labelPasswd,QCheckBox#cbkAuto {
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
	color: rgb(46, 44, 45);
}


QWidget#widgetPasswd, QWidget#widgetUser {
	border: 1px solid #333;
}

QLineEdit {
	border: none;
	background: transparent;
}

QComboBox{
	border: none;
}

</string>
  </property>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>156</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>4</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMin">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>4</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetPasswd" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>235</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelPasswd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>22</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwdEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>2</y>
      <width>190</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetUser" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>235</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelUsr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>22</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="serialEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>1</y>
      <width>193</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>235</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>登入</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbkAuto">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>記住登入</string>
   </property>
  </widget>
  <widget class="QMenuButton" name="btnConf">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>4</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMenuButton</class>
   <extends>QPushButton</extends>
   <header location="global">aide/common/qmenubutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialEdit</tabstop>
  <tabstop>passwdEdit</tabstop>
  <tabstop>cbkAuto</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnConf</tabstop>
  <tabstop>btnMin</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
